examples/virtual-machines/linux/basic-password/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "azurerm_resource_group" "main" {
+  name     = "${var.prefix}-resources"
+  location = var.location
+}
+
+resource "azurerm_virtual_network" "main" {
+  name                = "${var.prefix}-network"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${azurerm_resource_group.main.location}"
```

Comment:
should we use .12 syntax uniformly?

Reply:
@jackofallops has these done locally (he's also been adding examples but they'll go in a follow-up PR) - rather than conflict with that I'm going to leave this for the moment (since it's still 0.12 compatible)